spec/github_helper_spec.rb

```diff
@@ -19,4 +19,27 @@
       end
     end
   end
+
+  describe 'github_token' do
+    after do
+      ENV['GHHELPER_ACCESS'] = nil
+      ENV['GITHUB_TOKEN'] = nil
+    end
+
+    it 'can use `GHHELPER_ACCESS`' do
+      ENV['GHHELPER_ACCESS'] = 'GHHELPER_ACCESS'
+      expect(described_class.github_token).to eq('GHHELPER_ACCESS')
+    end
+
+    it 'can use `GITHUB_TOKEN`' do
+      ENV['GITHUB_TOKEN'] = 'GITHUB_TOKEN'
+      expect(described_class.github_token).to eq('GITHUB_TOKEN')
+    end
+
+    it 'prioritizes GHHELPER_ACCESS` over `GITHUB_TOKEN` if both are present' do
+      ENV['GITHUB_TOKEN'] = 'GITHUB_TOKEN'
+      ENV['GHHELPER_ACCESS'] = 'GHHELPER_ACCESS'
+      expect(described_class.github_token).to eq('GHHELPER_ACCESS')
+    end
+  end
```

Comment:
Please add a unit test for when neither of the 2 env vars are defined.

As per [my suggestion above](https://github.com/wordpress-mobile/release-toolkit/pull/313/files#r757899837), the test should check that `UI.user_error!` is called, and with the expected message, when this happens.

Reply:
Added with the check in 0374a55f0b81c2d5cec4e0a277991ffd7c6ae299